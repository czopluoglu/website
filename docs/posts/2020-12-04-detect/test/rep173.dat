11000110111110011001
00011101010111010100
00001000101011001110
11111101111000101111
11000001111100100011
10000000011111101100
10101100101110001100
10000010100110110011
01101010110000000111
10000011011101110100
11001110010011011111
10001001011000010110
00111001011011011010
10101101011000101110
11101000101010010011
11110100111011000110
00111011011110111110
00111001001011000000
00000000111010100011
11001100011011000100
10101100011111110111
00001101011000100011
00011101011000100110
11100001111000101100
10111001001000001110
11111110000000110101
00011001011001100010
11011111110000111010
10000001101110101011
10000101111110110111
10111011001111011101
10111000100010100111
11111001001010000011
11111011111110110111
00001000000110000011
10101001011110110001
01101001011101101011
01001001101101110110
11010001100010000110
11100011111111111011
11111111100011000111
11111001111111111111
00001100001000001111
11110111111111101111
10000001001100001011
00110001101011010001
01100001110010101100
11110011001111101111
00000000001110100100
00000001011011000111
11100010011010100011
01101011101110111101
10001000100011001110
11000001101011111111
11000101000010000111
10001111111000101111
11111100001010101010
10101001100010101011
11110001101111001111
11001011001110001111
01100101001111111111
00111110001100101111
11001000011010000100
10001010110000100110
01110011011110101111
10101001101111100111
11101001101000101010
10011101111111111010
00101111111011111111
11000010101111110011
10100101101110101011
10100000100101001001
11000101011011111110
10011101001000101000
11110011111011011111
11110110110010001110
01110010011010100110
10000000011001111011
00010101011111110110
11101011001111111011
11000001010010101010
11110001010110000010
10100000001111011110
01110001011111100111
01011111101010100110
10010000010111101111
11000100101110111001
01011001101100011111
11010100000111111110
10100000001111100101
00110111111111010111
01111000110011011110
11001001111011000101
01110011001100001111
11100110011011010011
00101111001111111111
11001111010001011010
01111111100001101110
00100000010011001001
11110010110010001001
11111111111111010001
11110111110110001001
10011111111110110000
00111001000011111010
10100111110010111100
01101011100100000110
01010001001100111110
01001101001000000000
11011111101011110110
00010111111111010110
11010000011111101011
11111111111010101111
10111110101000100010
10001000110011011111
00010100111010100110
11100001011000100010
00100101100110001111
10010111011110110011
10000010111100111110
11110101110110110101
11010111111111101110
10001101111010011111
00010101110000100110
10101110010010111111
10101111101101101111
11010000011110101111
00011000101100000001
00011100110111111111
11101000011110100101
00101110010010011010
10011001011110011011
10000000101111111010
10100101000000011110
01100010111111001111
00010101011000110010
00110000101100001001
10011110100011100000
01100110111011010111
00101001011101111111
00110111111110000010
10001100111111111110
10100001111000011110
11001000001100111111
00101100001101010110
10111111111111111110
01101100100000011111
00110000011011000010
10011100101000101110
10100101101100111010
11010011100000001011
00000000001010000011
11110000000111001011
01110001011000110100
11000101111000000110
11101001111101110110
10110111111100111111
10000000101010001001
01010111111110001010
10101111111110111111
10111101001001100111
01000000111010000011
10000001000011100100
00101000001000000010
01001000100001101011
11011101111010101100
01000011010000100000
11001101110011101111
10110111101111101011
11110011010000110010
00111011010111101010
01100000010010011001
00110001000000010110
01011011111100101010
00101000000000100110
11001001001100001010
11000001000010001011
10001011101011100110
00011110111010010110
10110010101101101111
00110001101110011011
10111111111011000010
11010101001111100111
00100101000100000010
10110001111010101111
01011111101000100010
01101111101111101010
11011101011011101101
11011110111011101100
00110000011011100000
11010001100000001111
11011001100001101100
10000000011100010110
11111111011111111000
10010000111011000010
11110001011010110011
00100011111110000110
10111001010010001110
11001001111000000001
01000110001001100001
11001111100110001011
11100111001010001010
11101101111110100011
11100110111111110011
00000001001111001010
01111011011111111111
00001001100110001001
00000000001011101100
10010000001000001010
00000101111011010111
10010010010111101111
11000100010011011110
11101111011101011111
11100100011101100101
01100001001111111011
11110101111101111110
00000001001010000110
01011111011010111010
11001111010100101011
10001110011111111001
00001001000101110111
11111111111001000110
11111001111110100110
11101010011011011011
10110011111010011110
00001100100111100011
00101001010000001010
11110011101111000000
10011100011111110110
10110000001101011110
00000001010111001011
11101111111100111111
00010000000000000010
11010101001010000001
10000000000010101010
00010000101011101110
10000101011010000110
01100011010011000110
11111011111111011111
11110100101111101111
10010011111111110010
11000011011101001111
11011001111111101111
10011011111101010111
11110101011100101111
10111010101001001111
10100100010010110111
11001001101010011110
11011111111111111111
11101100001110011110
10100111101100101111
