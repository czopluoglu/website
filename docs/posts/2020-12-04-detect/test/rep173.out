-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep173.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.40

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.24
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  5
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.6431
                   IDN index value:   0.6842
                           Ratio r:   0.5422

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    1    2    2    3    3    3    2    3    4
    5    5    4    4    5    4    1    4    2    1

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    2   17   20

   -----------CLUSTER  2-------------

    3    4    8   19

   -----------CLUSTER  3-------------

    5    6    7    9

   -----------CLUSTER  4-------------

   10   13   14   16   18

   -----------CLUSTER  5-------------

   11   12   15
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d+++-----+-+++---+-+
   +d+++-+---+++-------
   ++d++--++-------+-+-
   +++d+------+-+----+-
   -+++d+++--+---------
   ----+d+--++++++--+-+
   -+--++d++-+------++-
   --+-+-+d---+--+++---
   --+---+-d+++--+++--+
   +----+--+d+++--++--+
   -+--+++-++d++-------
   ++-+-+-++++d-+--+---
   ++---+---++-d+-+++-+
   +--+-+-----++d++--+-
   -----+-++----+d+++++
   -------+++--+++d+++-
   --+----+++-++-++d+--
   +----++-----+-+++d++
   --++--+------+++-+d+
   +----+--++--+-+--++d
 
 No cross validation for this DETECT run
